src/distrs/chisq.jl

```diff
@@ -11,3 +11,15 @@ import .RFunctions:
     chisqinvccdf,
     chisqinvlogcdf,
     chisqinvlogccdf
+
+# pdf for numbers with generic types
+function chisqpdf(k::Real, x::Number)
+  hk = 0.5 * k  # hlaf k
```

Comment:
Don't use `Float64` literals. Divide with `2` instead. The compiler will make it as efficient.